pkg/operator/encryption/operator_client_override.go

```diff
@@ -0,0 +1,33 @@
+package encryption
+
+import (
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/cluster-authentication-operator/pkg/controllers/common"
+	oauthapiconfigobserver "github.com/openshift/cluster-authentication-operator/pkg/operator/configobservation"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type UnsupportedConfigOverrideClient struct {
+	v1helpers.OperatorClient
+}
+
+func OperatorClientWithUnsupportedConfigOverride(delegate v1helpers.OperatorClient) v1helpers.OperatorClient {
+	return &UnsupportedConfigOverrideClient{OperatorClient: delegate}
+}
+
+func (op *UnsupportedConfigOverrideClient) GetOperatorState() (spec *operatorv1.OperatorSpec, status *operatorv1.OperatorStatus, resourceVersion string, err error) {
```

Comment:
simply override `GetOperatorState` that is used by the key controller to read the encryption config from `spec.UnsupportedConfigOverrides.Raw`